packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -96,6 +96,22 @@ export function getWebpackConfig(
         context: worker.repoRoot,
         manifest: DLL_MANIFEST,
       }),
+      // Fix for @n8n/json-schema-to-zod ESM imports without extensions
+      // This package uses ESM imports without file extensions, which webpack 5 requires
+      new webpack.NormalModuleReplacementPlugin(/^\.\.?\/.*$/, (resource: any) => {
```

Comment:
Did we consider creating a rule that unsets fullySpecified for this package?  It seems like this would be a more idiomatic approach:

https://webpack.js.org/configuration/resolve/#resolvefullyspecified
```
{
  test: ...,
  include: ..., 
  resolve: {
    fullySpecified: false
  },
},
```